bin/check-influxdb-metrics.rb

```diff
@@ -174,23 +174,25 @@ def encode_parameters(parameters)
   def today_metrics
     today_info = metrics(config[:metric], TODAY_START_PERIOD, today_end_period, false)
     @today_metric_count = validate_metrics_and_count(today_info)
-    value = if @today_metric_count > 0
-              series = read_series_from_metrics(today_info)
-              @today_metrics = store_metrics(series)
-              read_value_from_series(series)
-            end
-    value
+    if @today_metric_count > 0
```

Comment:
Better to use a nil check, otherwise if you have a use-case to check for negative values (which are perfectly valid) then it'll break